src/object/class_object.c

```diff
@@ -7932,14 +7959,14 @@ classobj_make_descriptor (MOP class_mop, SM_CLASS * classobj, SM_COMPONENT * com
  * |               | /UK(asc) | /UK(desc) |         |          |  /R-Idx   |
  * |               |          |   /R-UK   |         |          |           |
  * +---+-----------+----------+-----------+---------+----------+-----------+
- * |   |   PK(asc) |  share   |  new idx  |  error  |  error   |  new idx  |
+ * |   |   PK(asc) |  error   |  new idx  |  error  |  error   |  new idx  |
```

Comment:
trivial : PK(desc) / UK(desc) <== > PK(desc) / UK(desc) is 'share'. it seems to be rule for PK and UK.